spec/check_docs_spec.rb

```diff
@@ -0,0 +1,36 @@
+# encoding: utf-8
+require 'logstash-integration-kafka_jars'
+
+describe "[DOCS]" do
+
+  let(:docs_files) do
+    ['index.asciidoc', 'input-kafka.asciidoc', 'output-kafka.asciidoc'].map { |name| File.join('docs', name) }
+  end
+
+  let(:kafka_version_properties) do
+    loader = java.lang.Thread.currentThread.getContextClassLoader
+    version = loader.getResource('kafka/kafka-version.properties')
+    fail "kafka-version.properties missing" unless version
+    properties = java.util.Properties.new
+    properties.load version.openStream
+    properties
+  end
+
+  it 'is sync-ed with Kafka client version' do
+    version = kafka_version_properties.get('version') # e.g. '2.5.1'
+
+    fails = docs_files.map do |file|
+      if line = File.readlines(file).find { |line| line.index(':kafka_client:') }
+        puts "found #{line.inspect} in #{file}" if $VERBOSE # e.g. ":kafka_client: 2.5\n"
```

Comment:
@karenzone we load the Kafka client version used by the plugin and than simply check the 3 docs files (*'index.asciidoc', 'input-kafka.asciidoc', 'output-kafka.asciidoc'*) for  a `:kafka_client: ...` whether it matches the `MAJOR.MINOR` or the full version `MAJOR.MINOR.PATCH`. isn't perfect but still useful to not fall out-of-sync for the next person doing a Kafka upgrade.

Reply:
Thanks for adding this check.